advocacy_docs/supported-open-source/patroni/migrating.mdx

```diff
@@ -0,0 +1,323 @@
+systemctl enable patroni
+```
+
+5. Start the Patroni systemd unit. This is done so you start Patroni right now. Run:
+
+```bash
+systemctl start patroni
+```
+
+#### 3. Hand over Postgres startup to Patroni
+
+At this point Patroni is already monitoring the Postgres instances on each node. However, Postgres is still running using its own systemd unit (`postgres`). Ideally we should make Patroni start Postgres through its own means, and stop using the Postgres systemd unit.
+
+We recommend starting with the standbys, and then proceeding with the primary. Also, we assume you can immediately restart the standbys, and that you want to restart the primary during a maintenance window. The steps:
+
+1. Immediately restart of standby `postgres-node-2`. On any Postgres node:
+
+```bash
+patronictl -c /etc/patroni.yml restart test_cluster postgres-node-2 --scheduled now --force
+```
+
+2. Immediately restart of standby `postgres-node-3`. On any Postgres node:
+
+```bash
+patronictl -c /etc/patroni.yml restart test_cluster postgres-node-3 --scheduled now --force
+```
+
+3. Schedule a restart of primary `postgres-node-1` to occur during a maintenance window. On any Postgres node:
+
+```bash
+patronictl -c /etc/patroni.yml restart test_cluster postgres-node-1 --scheduled SOME_SCHEDULE --force
+```
+
+**Note:** `SOME_SCHEDULE` should be filled with the desired timestamp in an unambiguous format (e.g. `2023-07-25T20:09`).
+
+Once all Postgres instances were restarted, they will be already running through Patroni instead of through the Postgres systemd unit.
+
+#### 4. Clean up slots from original cluster
+
+At this point we assume all Postgres instances are already running through Patroni, as described in the previous section.
+
+As we intend to clean up slots from original cluster, before proceeding with the next steps you should confirm that both Postgres standby nodes have already received (and potentially consumed) all WAL files that exist between the replication slots that existed in the original cluster and the replication slots that were created by Patroni. In other words make sure that:
+
+1. `postgres-node-2` has received all WALs between the LSNs reported by these replication slots in `postgres-node-1`:
+    1. `slot_for_postgres_node_2` &mdash; previously existing slot in the original cluster.
+    2. `postgres_node_2` &mdash; slot automatically created by Patroni based on node name.
+2. `postgres-node-3` has received all WALs between the LSNs reported by these replication slots in `postgres-node-1`:
+    1. `slot_for_postgres_node_3` &mdash; previously existing slot in the original cluster.
+    2. `postgres_node_3` &mdash; slot automatically created by Patroni based on node name.
+
+**Note:** You can use a query like the one below to check the required LSN of these slots. You need to make sure that `restart_lsn` of the slots created by Patroni are `>=` the `restart_lsn` of the original slots. On node `postgres-node-1`, through a `psql` session:
```

Comment:
```suggestion
**Note:** You can use a query like the one below to check the required LSN of these slots. You need to make sure that the `restart_lsn` of the slots created by Patroni is `>=` then the `restart_lsn` of the original slots. On node `postgres-node-1`, through a `psql` session:
```

Reply:
I guess you meant `than` instead of `then`. Will fix it with a follow-up commit.